modules/parser/src/synchronizers/xviz-stream-buffer.js

```diff
@@ -332,23 +342,29 @@ export default class XVIZStreamBuffer {
     });
 
     for (const streamName in timeslice.streams) {
-      streams[streamName][index] = timeslice.streams[streamName];
+      let value = timeslice.streams[streamName];
+      if (value === null) {
+        // Explicitly delete a stream
+        delete timesliceAtInsertPosition.streams[streamName];
```

Comment:
Wait, we can incrementally delete a stream?  I don't understand this part.  When would an incremental update not add to existing data.

Reply:
In incremental mode you need to explicitly send a `null` to delete a stream, see https://github.com/uber/xviz/blob/jlisee/update-docs/docs/protocol-schema/session-protocol.md#state_update

Reply:
I still have append mode in my brain, my bad.